admin_guide/cluster_metrics.adoc

```diff
@@ -16,298 +16,382 @@ As an OpenShift administrator, you may want to view the metrics from all
+template parameters. If you wish, you can also specify any of the template parameters when calling these
+commands.
+
+=== Deploying with Persistent Storage
+
+The following command will set the Hawkular Metrics route to use `hawkular-metrics.example.com` and will
+be deployed using persistent storage.
+
+In order for this command to work, you must have a persistent volume of sufficient size available to be used.
+
+[options="nowrap"]
 ----
-apiVersion: "v1"
-kind: "List"
-items:
-  -
-    apiVersion: "v1"
-    kind: "Service"
-    metadata:
-      labels:
-        provider: "fabric8"
-        component: "grafana"
-      name: "grafana"
-    spec:
-      ports:
-        -
-          port: 80
-          targetPort: "http"
-      selector:
-        provider: "fabric8"
-        component: "grafana"
-  -
-    apiVersion: "v1"
-    kind: "ReplicationController"
-    metadata:
-      labels:
-        provider: "fabric8"
-        component: "grafana"
-      name: "grafana"
-    spec:
-      replicas: 1
-      selector:
-        provider: "fabric8"
-        component: "grafana"
-      template:
-        metadata:
-          labels:
-            provider: "fabric8"
-            component: "grafana"
-        spec:
-          containers:
-            -
-              env:
-                -
-                  name: "INFLUXDB_SERVICE_NAME"
-                  value: "INFLUXDB_MONITORING"
-                -
-                  name: "GRAFANA_DEFAULT_DASHBOARD"
-                  value: "/dashboard/file/kubernetes.json"
-              image: "fabric8/grafana:1.9.1_2"
-              name: "grafana"
-              ports:
-                -
-                  containerPort: 3000
-                  name: "http"
+$ oc process -f metrics.yaml -v \
+HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com \
+| oc create -f -
 ----
-====
 
-Save it to a file and create it with:
+=== Deploying without Persistent Storage
 
+The following command will set the Hawkular Metrics route to use `hawkular-metrics.example.com` and will
+be deployed without persistent storage.
+
+[IMPORTANT]
+====
+Since this is being deployed without persistent storage, metric data loss can occur.
+====
+
+[options="nowrap"]
 ----
-$ oc create -f <path/to/grafana.yaml>
+$ oc process -f metrics.yaml -v \
+HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,USE_PERSISTENT_STORAGE=false \
+| oc create -f -
 ----
+
+== Accessing Metrics in the Console
+
+With the following setup, you will be able to view and access metrics from within the OpenShift console.
+
+[IMPORTANT]
+====
+By default the setup uses self signed certificates for Hawkular Metrics, which can cause problems
+when the console accesses these metrics. You may need to access the Hawkular Metrics 
+route directly in your browser to manually accept the certificate 
+(ie _https://hawkular-metrics.example.com/hawkular/metrics_) or specify the `hawkular-metrics.pem`
+secret to use a certificate trusted by the browser.
+====
```

Comment:
Ah, I see you do warn about certificates down here... I'd suggest moving this up to where you talk about certificates above. It's too late if the admin didn't read ahead and already ran `oc process` / `oc create`.